docs/components/plugins/ImageSpotlight.tsx

```diff
@@ -15,7 +15,12 @@ export default function ImageSpotlight({ alt, src, style, containerClassName, ca
   return (
     <figure
       className={mergeClasses('text-center bg-subtle py-2.5 my-5 rounded-lg', containerClassName)}>
-      <LightboxImage src={src} alt={alt} style={style} className="inline rounded-md" />
+      <LightboxImage
+        src={src}
+        alt={alt}
+        style={style}
+        className="inline rounded-md transition duration-300 ease-in-out hover:opacity-80"
```

Comment:
```suggestion
        className="inline rounded-md transition-opacity duration-default ease-in-out hover:opacity-80"
```

Tailwind classes nit.